genesis_ledgers/default.json

```diff
@@ -0,0 +1,5 @@
+{
+  "ledger": {
+     "name": "testnet_postake"
+  }
```

Comment:
Is this enough for the daemon to start?

Reply:
Yes, it should be. I'm testing to be sure before I merge, but everything else should just default to the compile time constants. This ledger name maps to a precompiled ledger that is available to all daemons.

Reply:
Tested locally, and appears to work fine.